proto/process/agent.proto

```diff
@@ -250,6 +250,14 @@ message CollectorManifest {
 	repeated Manifest manifests = 5;
 }
 
+message CollectorManifestCRD {
+	CollectorManifest manifest = 1;
```

Comment:
I think **not** adding metadata here is easier because:
- no tight coupling to the agent
- no need for a larger refactoring which intermixes resourcesMetadata + resourcesManifests